controllers/cluster_controller_test.go

```diff
@@ -197,42 +202,44 @@ var _ = Describe("Cluster Reconciler", func() {
 		}, timeout).Should(BeTrue())
 	})
 
-	It("Should re-apply finalizers if removed", func() {
+	t.Run("Should re-apply finalizers if removed", func(t *testing.T) {
+		g := NewWithT(t)
+
 		// Setup
 		cluster := &clusterv1.Cluster{
 			ObjectMeta: metav1.ObjectMeta{
 				GenerateName: "test5-",
 				Namespace:    "default",
 			},
 		}
-		Expect(testEnv.Create(ctx, cluster)).To(BeNil())
+		g.Expect(testEnv.Create(ctx, cluster)).To(Succeed())
 		key := client.ObjectKey{Name: cluster.Name, Namespace: cluster.Namespace}
 		defer func() {
 			err := testEnv.Delete(ctx, cluster)
-			Expect(err).NotTo(HaveOccurred())
+			g.Expect(err).NotTo(HaveOccurred())
 		}()
 
 		// Wait for reconciliation to happen.
-		Eventually(func() bool {
+		g.Eventually(func() bool {
 			if err := testEnv.Get(ctx, key, cluster); err != nil {
 				return false
 			}
 			return len(cluster.Finalizers) > 0
 		}, timeout).Should(BeTrue())
 
 		// Remove finalizers
-		Eventually(func() bool {
+		g.Eventually(func() bool {
 			ph, err := patch.NewHelper(cluster, testEnv)
-			Expect(err).ShouldNot(HaveOccurred())
+			g.Expect(err).NotTo(HaveOccurred())
 			cluster.SetFinalizers([]string{})
-			Expect(ph.Patch(ctx, cluster, patch.WithStatusObservedGeneration{})).ShouldNot(HaveOccurred())
+			g.Expect(ph.Patch(ctx, cluster, patch.WithStatusObservedGeneration{})).NotTo(HaveOccurred())
```

Comment:
`To(Succeed())` reads better here

Reply:
It sure does--good catch! I replaced a few others like this with `To(Succeed())` but missed this one.